hamming_test.py is not run at all with bazel

See #1199

**Describe the bug**

The file is not run with bazel. When run with another test runner, it fails.

**Code to reproduce the issue**

See #1199

**Other info / logs**

```
(proof_of_concept_pytest) /mnt/c/Users/gdemarmi/Desktop/projects/addons $ pytest ./tensorflow_addons/metrics/hamming_test.py
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/pep8.py:110: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')
================================================================= test session starts ==================================================================
platform linux -- Python 3.7.5, pytest-5.0.1, py-1.8.0, pluggy-0.13.0
rootdir: /mnt/c/Users/gdemarmi/Desktop/projects/addons
plugins: arraydiff-0.3, cov-2.7.1, doctestplus-0.2.0, flake8-1.0.4, forked-1.0.2, openfiles-0.4.0, pep8-1.0.6, remotedata-0.3.2, xdist-1.27.0, typeguard-2.7.1
collected 7 items

tensorflow_addons/metrics/hamming_test.py ..FFFFs                                                                                                [100%]

======================================================================= FAILURES =======================================================================
_________________________________________________________ HammingMetricsTest.test_mc_4_classes _________________________________________________________

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f15102d2d50>
fn = <function BaseSession._do_run.<locals>._run_fn at 0x7f1510306b90>
args = ({}, [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510277450> >], [], None, None)
message = 'Container localhost does not exist. (Could not find resource: localhost/total)\n\t [[{{node AssignAddVariableOp}}]]'
m = <re.Match object; span=(81, 111), match='[[{{node AssignAddVariableOp}}'>

    def _do_call(self, fn, *args):
      try:
>       return fn(*args)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1367:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

feed_dict = {}, fetch_list = [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510277450> >]
target_list = [], options = None, run_metadata = None

    def _run_fn(feed_dict, fetch_list, target_list, options, run_metadata):
      # Ensure any changes to the graph are reflected in the runtime.
      self._extend_graph()
      return self._call_tf_sessionrun(options, feed_dict, fetch_list,
>                                     target_list, run_metadata)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1352:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f15102d2d50>, options = None, feed_dict = {}
fetch_list = [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510277450> >]
target_list = [], run_metadata = None

    def _call_tf_sessionrun(self, options, feed_dict, fetch_list, target_list,
                            run_metadata):
      return tf_session.TF_SessionRun_wrapper(self._session, options, feed_dict,
                                              fetch_list, target_list,
>                                             run_metadata)
E     tensorflow.python.framework.errors_impl.NotFoundError: Container localhost does not exist. (Could not find resource: localhost/total)
E        [[{{node AssignAddVariableOp}}]]

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1445: NotFoundError

During handling of the above exception, another exception occurred:

self = <tensorflow_addons.metrics.hamming_test.HammingMetricsTest testMethod=test_mc_4_classes>, args = (), kwargs = {}

    def decorated(self, *args, **kwargs):
      try:
        with context.graph_mode():
          with self.test_session(use_gpu=use_gpu, config=config):
>           f(self, *args, **kwargs)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:1111:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tensorflow_addons/metrics/hamming_test.py:70: in test_mc_4_classes
    self.update_obj_states(hl_obj, actuals, predictions)
tensorflow_addons/metrics/hamming_test.py:37: in update_obj_states
    self.evaluate(update_op)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:2061: in evaluate
    return sess.run(tensors)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:1638: in run
    return super(ErrorLoggingSession, self).run(*args, **kwargs)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:960: in run
    run_metadata_ptr)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1183: in _run
    feed_dict_tensor, options, run_metadata)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1361: in _do_run
    run_metadata)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f15102d2d50>
fn = <function BaseSession._do_run.<locals>._run_fn at 0x7f1510306b90>
args = ({}, [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510277450> >], [], None, None)
message = 'Container localhost does not exist. (Could not find resource: localhost/total)\n\t [[{{node AssignAddVariableOp}}]]'
m = <re.Match object; span=(81, 111), match='[[{{node AssignAddVariableOp}}'>

    def _do_call(self, fn, *args):
      try:
        return fn(*args)
      except errors.OpError as e:
        message = compat.as_text(e.message)
        m = BaseSession._NODEDEF_NAME_RE.search(message)
        node_def = None
        op = None
        if m is not None:
          node_name = m.group(3)
          try:
            op = self._graph.get_operation_by_name(node_name)
            node_def = op.node_def
          except KeyError:
            pass
        message = error_interpolation.interpolate(message, self._graph)
        if 'only supports NHWC tensor format' in message:
          message += ('\nA possible workaround: Try disabling Grappler optimizer'
                      '\nby modifying the config for creating the session eg.'
                      '\nsession_config.graph_options.rewrite_options.'
                      'disable_meta_optimizer = True')
>       raise type(e)(node_def, op, message)
E       tensorflow.python.framework.errors_impl.NotFoundError: Container localhost does not exist. (Could not find resource: localhost/total)
E                [[{{node AssignAddVariableOp}}]]

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1386: NotFoundError
----------------------------------------------------------------- Captured stderr call -----------------------------------------------------------------
2020-03-02 15:06:31.839236: W tensorflow/core/framework/op_kernel.cc:1655] OP_REQUIRES failed at resource_variable_ops.cc:540 : Not found: Container localhost does not exist. (Could not find resource: localhost/total)
E0302 15:06:31.839798 139730329732928 test_util.py:1644] Container localhost does not exist. (Could not find resource: localhost/total)
         [[{{node AssignAddVariableOp}}]]
------------------------------------------------------------------ Captured log call -------------------------------------------------------------------
ERROR    tensorflow:test_util.py:1644 Container localhost does not exist. (Could not find resource: localhost/total)
                                         [[{{node AssignAddVariableOp}}]]
_________________________________________________________ HammingMetricsTest.test_mc_5_classes _________________________________________________________

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f15101d6e10>
fn = <function BaseSession._do_run.<locals>._run_fn at 0x7f15101ea0e0>
args = ({}, [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510277c60> >], [], None, None)
message = 'Container localhost does not exist. (Could not find resource: localhost/total)\n\t [[{{node AssignAddVariableOp}}]]'
m = <re.Match object; span=(81, 111), match='[[{{node AssignAddVariableOp}}'>

    def _do_call(self, fn, *args):
      try:
>       return fn(*args)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1367:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

feed_dict = {}, fetch_list = [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510277c60> >]
target_list = [], options = None, run_metadata = None

    def _run_fn(feed_dict, fetch_list, target_list, options, run_metadata):
      # Ensure any changes to the graph are reflected in the runtime.
      self._extend_graph()
      return self._call_tf_sessionrun(options, feed_dict, fetch_list,
>                                     target_list, run_metadata)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1352:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f15101d6e10>, options = None, feed_dict = {}
fetch_list = [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510277c60> >]
target_list = [], run_metadata = None

    def _call_tf_sessionrun(self, options, feed_dict, fetch_list, target_list,
                            run_metadata):
      return tf_session.TF_SessionRun_wrapper(self._session, options, feed_dict,
                                              fetch_list, target_list,
>                                             run_metadata)
E     tensorflow.python.framework.errors_impl.NotFoundError: Container localhost does not exist. (Could not find resource: localhost/total)
E        [[{{node AssignAddVariableOp}}]]

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1445: NotFoundError

During handling of the above exception, another exception occurred:

self = <tensorflow_addons.metrics.hamming_test.HammingMetricsTest testMethod=test_mc_5_classes>, args = (), kwargs = {}

    def decorated(self, *args, **kwargs):
      try:
        with context.graph_mode():
          with self.test_session(use_gpu=use_gpu, config=config):
>           f(self, *args, **kwargs)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:1111:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tensorflow_addons/metrics/hamming_test.py:105: in test_mc_5_classes
    self.update_obj_states(hl_obj, actuals, predictions)
tensorflow_addons/metrics/hamming_test.py:37: in update_obj_states
    self.evaluate(update_op)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:2061: in evaluate
    return sess.run(tensors)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:1638: in run
    return super(ErrorLoggingSession, self).run(*args, **kwargs)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:960: in run
    run_metadata_ptr)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1183: in _run
    feed_dict_tensor, options, run_metadata)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1361: in _do_run
    run_metadata)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f15101d6e10>
fn = <function BaseSession._do_run.<locals>._run_fn at 0x7f15101ea0e0>
args = ({}, [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510277c60> >], [], None, None)
message = 'Container localhost does not exist. (Could not find resource: localhost/total)\n\t [[{{node AssignAddVariableOp}}]]'
m = <re.Match object; span=(81, 111), match='[[{{node AssignAddVariableOp}}'>

    def _do_call(self, fn, *args):
      try:
        return fn(*args)
      except errors.OpError as e:
        message = compat.as_text(e.message)
        m = BaseSession._NODEDEF_NAME_RE.search(message)
        node_def = None
        op = None
        if m is not None:
          node_name = m.group(3)
          try:
            op = self._graph.get_operation_by_name(node_name)
            node_def = op.node_def
          except KeyError:
            pass
        message = error_interpolation.interpolate(message, self._graph)
        if 'only supports NHWC tensor format' in message:
          message += ('\nA possible workaround: Try disabling Grappler optimizer'
                      '\nby modifying the config for creating the session eg.'
                      '\nsession_config.graph_options.rewrite_options.'
                      'disable_meta_optimizer = True')
>       raise type(e)(node_def, op, message)
E       tensorflow.python.framework.errors_impl.NotFoundError: Container localhost does not exist. (Could not find resource: localhost/total)
E                [[{{node AssignAddVariableOp}}]]

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1386: NotFoundError
----------------------------------------------------------------- Captured stderr call -----------------------------------------------------------------
2020-03-02 15:06:32.960872: W tensorflow/core/framework/op_kernel.cc:1655] OP_REQUIRES failed at resource_variable_ops.cc:540 : Not found: Container localhost does not exist. (Could not find resource: localhost/total)
E0302 15:06:32.961307 139730329732928 test_util.py:1644] Container localhost does not exist. (Could not find resource: localhost/total)
         [[{{node AssignAddVariableOp}}]]
------------------------------------------------------------------ Captured log call -------------------------------------------------------------------
ERROR    tensorflow:test_util.py:1644 Container localhost does not exist. (Could not find resource: localhost/total)
                                         [[{{node AssignAddVariableOp}}]]
_________________________________________________________ HammingMetricsTest.test_ml_4_classes _________________________________________________________

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f151ea9bdd0>
fn = <function BaseSession._do_run.<locals>._run_fn at 0x7f1510376710>
args = ({}, [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510398e40> >], [], None, None)
message = 'Container localhost does not exist. (Could not find resource: localhost/total)\n\t [[{{node AssignAddVariableOp}}]]'
m = <re.Match object; span=(81, 111), match='[[{{node AssignAddVariableOp}}'>

    def _do_call(self, fn, *args):
      try:
>       return fn(*args)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1367:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

feed_dict = {}, fetch_list = [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510398e40> >]
target_list = [], options = None, run_metadata = None

    def _run_fn(feed_dict, fetch_list, target_list, options, run_metadata):
      # Ensure any changes to the graph are reflected in the runtime.
      self._extend_graph()
      return self._call_tf_sessionrun(options, feed_dict, fetch_list,
>                                     target_list, run_metadata)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1352:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f151ea9bdd0>, options = None, feed_dict = {}
fetch_list = [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510398e40> >]
target_list = [], run_metadata = None

    def _call_tf_sessionrun(self, options, feed_dict, fetch_list, target_list,
                            run_metadata):
      return tf_session.TF_SessionRun_wrapper(self._session, options, feed_dict,
                                              fetch_list, target_list,
>                                             run_metadata)
E     tensorflow.python.framework.errors_impl.NotFoundError: Container localhost does not exist. (Could not find resource: localhost/total)
E        [[{{node AssignAddVariableOp}}]]

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1445: NotFoundError

During handling of the above exception, another exception occurred:

self = <tensorflow_addons.metrics.hamming_test.HammingMetricsTest testMethod=test_ml_4_classes>, args = (), kwargs = {}

    def decorated(self, *args, **kwargs):
      try:
        with context.graph_mode():
          with self.test_session(use_gpu=use_gpu, config=config):
>           f(self, *args, **kwargs)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:1111:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tensorflow_addons/metrics/hamming_test.py:124: in test_ml_4_classes
    self.update_obj_states(hl_obj, actuals, predictions)
tensorflow_addons/metrics/hamming_test.py:37: in update_obj_states
    self.evaluate(update_op)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:2061: in evaluate
    return sess.run(tensors)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:1638: in run
    return super(ErrorLoggingSession, self).run(*args, **kwargs)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:960: in run
    run_metadata_ptr)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1183: in _run
    feed_dict_tensor, options, run_metadata)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1361: in _do_run
    run_metadata)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f151ea9bdd0>
fn = <function BaseSession._do_run.<locals>._run_fn at 0x7f1510376710>
args = ({}, [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f1510398e40> >], [], None, None)
message = 'Container localhost does not exist. (Could not find resource: localhost/total)\n\t [[{{node AssignAddVariableOp}}]]'
m = <re.Match object; span=(81, 111), match='[[{{node AssignAddVariableOp}}'>

    def _do_call(self, fn, *args):
      try:
        return fn(*args)
      except errors.OpError as e:
        message = compat.as_text(e.message)
        m = BaseSession._NODEDEF_NAME_RE.search(message)
        node_def = None
        op = None
        if m is not None:
          node_name = m.group(3)
          try:
            op = self._graph.get_operation_by_name(node_name)
            node_def = op.node_def
          except KeyError:
            pass
        message = error_interpolation.interpolate(message, self._graph)
        if 'only supports NHWC tensor format' in message:
          message += ('\nA possible workaround: Try disabling Grappler optimizer'
                      '\nby modifying the config for creating the session eg.'
                      '\nsession_config.graph_options.rewrite_options.'
                      'disable_meta_optimizer = True')
>       raise type(e)(node_def, op, message)
E       tensorflow.python.framework.errors_impl.NotFoundError: Container localhost does not exist. (Could not find resource: localhost/total)
E                [[{{node AssignAddVariableOp}}]]

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1386: NotFoundError
----------------------------------------------------------------- Captured stderr call -----------------------------------------------------------------
2020-03-02 15:06:33.279781: W tensorflow/core/framework/op_kernel.cc:1655] OP_REQUIRES failed at resource_variable_ops.cc:540 : Not found: Container localhost does not exist. (Could not find resource: localhost/total)
E0302 15:06:33.280279 139730329732928 test_util.py:1644] Container localhost does not exist. (Could not find resource: localhost/total)
         [[{{node AssignAddVariableOp}}]]
------------------------------------------------------------------ Captured log call -------------------------------------------------------------------
ERROR    tensorflow:test_util.py:1644 Container localhost does not exist. (Could not find resource: localhost/total)
                                         [[{{node AssignAddVariableOp}}]]
_________________________________________________________ HammingMetricsTest.test_ml_5_classes _________________________________________________________

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f15102c8c90>
fn = <function BaseSession._do_run.<locals>._run_fn at 0x7f1510376f80>
args = ({}, [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f15103bda50> >], [], None, None)
message = 'Container localhost does not exist. (Could not find resource: localhost/total)\n\t [[{{node AssignAddVariableOp}}]]'
m = <re.Match object; span=(81, 111), match='[[{{node AssignAddVariableOp}}'>

    def _do_call(self, fn, *args):
      try:
>       return fn(*args)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1367:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

feed_dict = {}, fetch_list = [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f15103bda50> >]
target_list = [], options = None, run_metadata = None

    def _run_fn(feed_dict, fetch_list, target_list, options, run_metadata):
      # Ensure any changes to the graph are reflected in the runtime.
      self._extend_graph()
      return self._call_tf_sessionrun(options, feed_dict, fetch_list,
>                                     target_list, run_metadata)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1352:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f15102c8c90>, options = None, feed_dict = {}
fetch_list = [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f15103bda50> >]
target_list = [], run_metadata = None

    def _call_tf_sessionrun(self, options, feed_dict, fetch_list, target_list,
                            run_metadata):
      return tf_session.TF_SessionRun_wrapper(self._session, options, feed_dict,
                                              fetch_list, target_list,
>                                             run_metadata)
E     tensorflow.python.framework.errors_impl.NotFoundError: Container localhost does not exist. (Could not find resource: localhost/total)
E        [[{{node AssignAddVariableOp}}]]

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1445: NotFoundError

During handling of the above exception, another exception occurred:

self = <tensorflow_addons.metrics.hamming_test.HammingMetricsTest testMethod=test_ml_5_classes>, args = (), kwargs = {}

    def decorated(self, *args, **kwargs):
      try:
        with context.graph_mode():
          with self.test_session(use_gpu=use_gpu, config=config):
>           f(self, *args, **kwargs)

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:1111:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tensorflow_addons/metrics/hamming_test.py:158: in test_ml_5_classes
    self.update_obj_states(hl_obj, actuals, predictions)
tensorflow_addons/metrics/hamming_test.py:37: in update_obj_states
    self.evaluate(update_op)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:2061: in evaluate
    return sess.run(tensors)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/test_util.py:1638: in run
    return super(ErrorLoggingSession, self).run(*args, **kwargs)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:960: in run
    run_metadata_ptr)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1183: in _run
    feed_dict_tensor, options, run_metadata)
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1361: in _do_run
    run_metadata)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tensorflow.python.framework.test_util.ErrorLoggingSession object at 0x7f15102c8c90>
fn = <function BaseSession._do_run.<locals>._run_fn at 0x7f1510376f80>
args = ({}, [<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f15103bda50> >], [], None, None)
message = 'Container localhost does not exist. (Could not find resource: localhost/total)\n\t [[{{node AssignAddVariableOp}}]]'
m = <re.Match object; span=(81, 111), match='[[{{node AssignAddVariableOp}}'>

    def _do_call(self, fn, *args):
      try:
        return fn(*args)
      except errors.OpError as e:
        message = compat.as_text(e.message)
        m = BaseSession._NODEDEF_NAME_RE.search(message)
        node_def = None
        op = None
        if m is not None:
          node_name = m.group(3)
          try:
            op = self._graph.get_operation_by_name(node_name)
            node_def = op.node_def
          except KeyError:
            pass
        message = error_interpolation.interpolate(message, self._graph)
        if 'only supports NHWC tensor format' in message:
          message += ('\nA possible workaround: Try disabling Grappler optimizer'
                      '\nby modifying the config for creating the session eg.'
                      '\nsession_config.graph_options.rewrite_options.'
                      'disable_meta_optimizer = True')
>       raise type(e)(node_def, op, message)
E       tensorflow.python.framework.errors_impl.NotFoundError: Container localhost does not exist. (Could not find resource: localhost/total)
E                [[{{node AssignAddVariableOp}}]]

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/client/session.py:1386: NotFoundError
----------------------------------------------------------------- Captured stderr call -----------------------------------------------------------------
2020-03-02 15:06:33.498919: W tensorflow/core/framework/op_kernel.cc:1655] OP_REQUIRES failed at resource_variable_ops.cc:540 : Not found: Container localhost does not exist. (Could not find resource: localhost/total)
E0302 15:06:33.499402 139730329732928 test_util.py:1644] Container localhost does not exist. (Could not find resource: localhost/total)
         [[{{node AssignAddVariableOp}}]]
------------------------------------------------------------------ Captured log call -------------------------------------------------------------------
ERROR    tensorflow:test_util.py:1644 Container localhost does not exist. (Could not find resource: localhost/total)
                                         [[{{node AssignAddVariableOp}}]]
=================================================================== warnings summary ===================================================================
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/google/protobuf/descriptor.py:47
  /home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/google/protobuf/descriptor.py:47: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from google.protobuf.pyext import _message

/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/botocore/vendored/requests/packages/urllib3/_collections.py:1
/home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/botocore/vendored/requests/packages/urllib3/_collections.py:1
  /home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/botocore/vendored/requests/packages/urllib3/_collections.py:1: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    from collections import Mapping, MutableMapping

tensorflow_addons/metrics/hamming_test.py::HammingMetricsTest::test_keras_model
  /home/gdemarmi/softwares/python/anaconda/lib/python3.7/site-packages/tensorflow_core/python/framework/indexed_slices.py:348: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    if not isinstance(values, collections.Sequence):

-- Docs: https://docs.pytest.org/en/latest/warnings.html
============================================== 4 failed, 2 passed, 1 skipped, 4 warnings in 18.26 seconds ==============================================
```

@ssaishruthi could you take a look? Thanks!

--- a/tensorflow_addons/metrics/tests/hamming_test.py
+++ b/tensorflow_addons/metrics/tests/hamming_test.py
@@ -32,7 +32,6 @@ def check_results(obj, value):
     np.testing.assert_allclose(value, obj.result().numpy(), atol=1e-5)
 
 
-@pytest.mark.xfail(reason="See https://github.com/tensorflow/addons/issues/1200")
 def test_mc_4_classes():
     actuals = tf.constant(
         [
@@ -65,7 +64,6 @@ def test_mc_4_classes():
     check_results(hl_obj, 0.2857143)
 
 
-@pytest.mark.xfail(reason="See https://github.com/tensorflow/addons/issues/1200")
 def test_mc_5_classes():
     actuals = tf.constant(
         [
@@ -101,7 +99,6 @@ def test_mc_5_classes():
     check_results(hl_obj, 0.25)
 
 
-@pytest.mark.xfail(reason="See https://github.com/tensorflow/addons/issues/1200")
 def test_ml_4_classes():
     actuals = tf.constant([[1, 0, 1, 0], [0, 1, 0, 1], [0, 0, 0, 1]], dtype=tf.float32)
     predictions = tf.constant(
@@ -115,7 +112,6 @@ def test_ml_4_classes():
     check_results(hl_obj, 0.16666667)
 
 
-@pytest.mark.xfail(reason="See https://github.com/tensorflow/addons/issues/1200")
 def test_ml_5_classes():
     actuals = tf.constant(
         [

--- a/tensorflow_addons/metrics/tests/hamming_test.py
+++ b/tensorflow_addons/metrics/tests/hamming_test.py
@@ -15,7 +15,6 @@
 """Tests Hamming metrics."""
 
 import numpy as np
-import pytest
 import tensorflow as tf
 from tensorflow.keras import layers
 

